src/_playground/App.scss

```diff
@@ -113,7 +113,7 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
                 transition: background-color .25s;
             }
 
-            .frDocs-markdown {
+            .frDocs-markdown, .frDocs-Content__description {
```

Comment:
Do we need another class or just add the `frDocs-markdown` class to the `ReactMarkdown` component in `Description`?